net/transmission-web-control/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=transmission-web-control
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/ronggang/transmission-web-control
+PKG_SOURCE_DATE:=2019-04-16
+PKG_SOURCE_VERSION:=a747a87babb246b0d3e6dd108ad7b4c19462cfe6
+PKG_MIRROR_HASH:=90a2e33bd799a13504881e26980823fa181b47918900fc4ef03ae3daf0625f14
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Richard Yu <yurichard3839@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/transmission-web-control
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=BitTorrent
+  DEPENDS:=@(PACKAGE_transmission-daemon-openssl||PACKAGE_transmission-daemon-mbedtls)
+  CONFLICTS:=transmission-web
+  TITLE:=Transmission Web Control
+  URL:=https://github.com/ronggang/transmission-web-control
+  PKGARCH:=all
+endef
+
+define Package/transmission-web-control/description
+ Transmission Web Control is a custom Web UI for Transmission.
+endef
+
+define Build/Compile
+endef
+
+define Package/transmission-web-control/install
+	$(INSTALL_DIR) $(1)/usr/share/transmission/web
+	$(CP) $(PKG_BUILD_DIR)/src/* $(1)/usr/share/transmission/web
```

Comment:
Could probably be replaced with INSTALL_DATA

Reply:
It seems that INSTALL_DATA can't do recursive copy.